dhcp/server.go

```diff
@@ -95,6 +90,17 @@ func (h *DHCPHandler) fillPXE() []byte {
 
 //
 func (h *DHCPHandler) ServeDHCP(p dhcp4.Packet, msgType dhcp4.MessageType, options dhcp4.Options) (d dhcp4.Packet) {
+	dns, err := h.datasource.DNSAddresses()
+	if err == nil {
+		h.dhcpOptions = dhcp4.Options{
```

Comment:
No, I guess not. dhcp4.Options fields are not exposed and therefore not modifiable. The only way to give it a new option for dns seems to be re-instantiating it with the new option 6 []byte.